LXStudio-IDE/src/main/java/titanicsend/util/PanelStriper.java

```diff
@@ -0,0 +1,145 @@
+package titanicsend.util;
+
+import heronarts.lx.model.LXPoint;
+import titanicsend.model.TEVertex;
+
+import java.util.*;
+
+import static java.lang.Math.cos;
+
+public class PanelStriper {
+  public static final int MARGIN = 75000; // 75k microns ~= 3 inches
+  public static final int DISTANCE_BETWEEN_PIXELS = 50000; // 25k microns ~= 1 inch
+
+  public static List<LXPoint> stripe(TEVertex v0, TEVertex v1, TEVertex v2) {
+    FloorTransform floorTransform = new FloorTransform(v0, v1, v2);
+
+    List<FloorPoint> floorPoints = stripeFloor(
+            floorTransform.f0, floorTransform.f1, floorTransform.f2);
+
+    List<LXPoint> rv = new ArrayList<LXPoint>();
+    for (FloorPoint f : floorPoints) {
+      rv.add(floorTransform.fly(f));
+    }
+
+    return rv;
+  }
+
+  private static List<FloorPoint> stripeFloor(FloorPoint f0, FloorPoint f1, FloorPoint f2) {
```

Comment:
may be useful to have a level description of the approach here in a comment

Reply:
Agreed. Will work on that in the next revision.